File: app/src/main/java/cc/smartcash/wallet/Models/SendPayment.java
package cc.smartcash.wallet.Models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.google.gson.annotations.SerializedName;

@JsonIgnoreProperties(ignoreUnknown = true)
public class SendPayment {
    @SerializedName("FromAddress")
    private String fromAddress;
    @SerializedName("ToAddress")
    private String toAddress;
    @SerializedName("Amount")
    private Double amount;
    @SerializedName("UserKey")
    private String userKey;
    @SerializedName("code")
    private String code;
    @SerializedName("Email")
    private String email;

    public String getFromAddress() {
        return fromAddress;
    }

    public void setFromAddress(String fromAddress) {
        this.fromAddress = fromAddress;
    }

    public String getToAddress() {
        return toAddress;
    }

    public void setToAddress(String toAddress) {
        this.toAddress = toAddress;
    }

    public Double getAmount() {
        return amount;
    }

    public void setAmount(Double amount) {
        this.amount = amount;
    }

    public String getUserKey() {
        return userKey;
    }

    public void setUserKey(String userKey) {
        this.userKey = userKey;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}

File: app/src/main/java/cc/smartcash/wallet/Models/SendPayment.kt
package cc.smartcash.wallet.Models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.google.gson.annotations.SerializedName
import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
class SendPayment : Serializable {
    @SerializedName("FromAddress")
    var fromAddress: String? = null
    @SerializedName("ToAddress")
    var toAddress: String? = null
    @SerializedName("Amount")
    var amount: Double? = null
    @SerializedName("UserKey")
    var userKey: String? = null
    @SerializedName("code")
    var code: String? = null
    @SerializedName("Email")
    var email: String? = null
}